packages/mobile/src/escrow/saga.ts

```diff
@@ -184,11 +101,11 @@ function* transferStableTokenToEscrowWithoutCode(action: EscrowTransferPaymentAc
     }
 
     // Approve a transfer of funds to the Escrow contract.
-    Logger.debug(TAG + '@transferToEscrowWithoutCode', 'Approving escrow transfer')
-    const convertedAmount = contractKit.connection.web3.utils.toWei(amount.toString())
+    Logger.debug(TAG + '@transferToEscrow', 'Approving escrow transfer')
+    const convertedAmount = contractKit.connection.web3.utils.toWei(amount.toFixed(WEI_DECIMALS))
```

Comment:
When doing the math to convert to cEUR the amount ended up with decimals, so this call was failing because it allows a max of 18 decimals. Not sure why this doesn't affect the cUSD conversion (maybe just luck? I'm not sure this is not happening in the wild right now) but for regular sending we do `new BigNumber(amount).multipliedBy(decimalFactor).toFixed(0)` so it doesn't have the extra decimals issue.

As an example, when inviting someone by sending them R$0.10, I see this:
amount.toString(): 0.01674789114741644519 (20 decimals)
amount.toFixed(18): 0.016747891147416445
So the first one fails when passed as parameter to the `toWei` function.